joomla/joomla-cms
# [4.0] Language library prepared statements

### Summary of Changes

Adds some prepared statements and cleans up queries in `Joomla\CMS\Language` namespace.

### Testing Instructions

Set up a multilingual site, install multilingual sample data.
View an article in frontend, check that language switcher shows correct associations.

### Expected result

Works like before.

### Documentation Changes Required

No.
